portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,9 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
```

Comment:
Do you need these imports?

Reply:
oops.. this is for the next step, is that okay or should i delete this for now... and add it again later

Reply:
Thanks for clarifying :) Yea we usually add the imports when we use them so please delete